libs/core-ui/src/lib/util/StatisticsUtils.ts

```diff
@@ -8,10 +8,28 @@ import { ModelTypes } from "../Interfaces/IExplanationContext";
 import { ClassificationEnum, JointDataset } from "./JointDataset";
 
 export interface ILabeledStatistic {
+  key: string;
   label: string;
   stat: number;
 }
 
+export enum BinaryClassificationMetrics {
+  Accuracy = "accuracy",
+  Precision = "precision",
+  Recall = "recall",
+  FalseNegativeRate = "falseNegativeRate",
+  FalsePositiveRate = "falsePositiveRate",
+  SelectionRate = "selectionRate",
+  F1Score = "f1Score"
+}
+
+export enum RegressionMetrics {
+  MeanSquaredError = "meanSquaredError",
+  MeanAbsoluteError = "meanAbsoluteError",
+  MeanPrediction = "meanPrediction",
+  RSquared = "rSquared"
```

Comment:
I sometimes see this called as coefficient of determination:
https://github.com/microsoft/responsible-ai-toolbox/blob/775c2f352ece316a3c575016b00d16d6a680f7b5/libs/localization/src/lib/en.json#L1044
I wonder if we can make the ^2 look like squared using unicode, or use the other term instead, any thoughts about that?

Reply:
maybe you can use this ^2 unicode character: R²

Reply:
I think this comment applies to the en.json instead

Reply:
I'll see if I can make it work! Thanks for the suggestion

Reply:
added superscript 2. Using the unicode encoding somehow didn't work, but just copying the character in did.

Reply:
![image](https://user-images.githubusercontent.com/10245648/151447886-ec7af76c-13e3-462c-84ac-2c78d7a7238c.png)